src/drawers/drawer.ts

```diff
@@ -111,16 +111,22 @@ export module Drawers {
      *
      * @returns {d3.Selection} the renderArea selection
      */
-    public _getRenderArea() {
+    public renderArea() {
       return this._renderArea;
     }
 
-    public _getSelector(): string {
+    /**
+     * Returns the selector for this Drawer's visual elements.
+     */
+    public selector() {
```

Comment:
In fact trying to use the selector `""` already throws an error, but a really vague one, so throwing our own would be better (or returning `null`). I'll do that.